src/components/Content/HomePageBlurb.js

```diff
@@ -40,17 +39,17 @@ const styles = theme => ({
         whiteSpace: "nowrap"
     },
     headline: {
-        paddingLeft: theme.spacing.unit * 4,
-        paddingRight: theme.spacing.unit * 4,
-        marginTop: theme.spacing.unit,
+        paddingLeft: theme.spacing(4),
+        paddingRight: theme.spacing(4),
+        marginTop: theme.spacing(4),
```

Comment:
Do we want to add a spacing of 4 units to the top , because originally it wasn't

Reply:
Yeah it wasnt centered enough.